nes/config.py

```
@@ -36,44 +32,54 @@ def get_db_path(cls, override_path: Optional[str] = None) -> Path:
         database_url = os.getenv("NES_DB_URL")
-        if database_url:
-            parsed = urlparse(database_url)
-            return parsed.scheme
-        return "file"
+        if not database_url:
+            raise ValueError(
+                "NES_DB_URL environment variable must be set. "
+                "The database is managed separately from the codebase. Examples:\n"
+                "  file:///absolute/path/to/nes-db/v2\n"
+                "  file+memcached:///absolute/path/to/nes-db/v2"
+            )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)